licenses/more-licences.md

```diff
@@ -106,16 +106,43 @@
 > - **POM License**: Apache License, Version 2.0 - [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0)
 > - **POM License**: The 3-Clause BSD License - [https://opensource.org/licenses/BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause)
 
+## Creative Commons Legal Code
+
+**24** **Group:** `org.hdrhistogram` **Name:** `HdrHistogram` **Version:** `2.1.12` 
```

Comment:
This introduces a new license "2-clause bsd licence" for which we need to provide a copy, I have opened https://github.com/JonasKunz/elastic-otel-java/pull/1 to cover this.

However for the "Creative Commons Legal Code" or the "PUBLIC DOMAIN" I don't know what we should do with those.

Reply:
After checking a bit further, this code is dual-licensed and I've updated the PR to cover both.

Reply:
Thanks, merged!